### ISSUE TITLE
AG: unable to process any of the UMLS ontologies with AllegroGraph

### ISSUE THREAD

**Issue by alexskr** [ncbo/bioportal-project#215]
UMLS ontologies are not getting processed with AllegroGraph backend.

```
I, [2021-06-18T00:27:34.759051 #130438]  INFO -- : ["Triples /srv/ontoportal/data/repository/STY/1/umls_semantictypes.ttl appended in <http://data.bioontology.org/ontologies/STY/submissions/1>"]
E, [2021-06-18T00:27:34.831600 #130438] ERROR -- : ["Empty page encountered. Retrying 1 times..."]
E, [2021-06-18T00:27:36.834615 #130438] ERROR -- : ["Empty page encountered. Retrying 2 times..."]
E, [2021-06-18T00:27:38.837624 #130438] ERROR -- : ["Empty page encountered. Retrying 3 times..."]
E, [2021-06-18T00:27:40.840652 #130438] ERROR -- : ["Empty page encountered. Retrying 4 times..."]
E, [2021-06-18T00:27:42.843647 #130438] ERROR -- : ["Empty page encountered. Retrying 5 times..."]
E, [2021-06-18T00:27:44.844710 #130438] ERROR -- : ["Empty page encountered. Retrying 6 times..."]
E, [2021-06-18T00:27:46.847908 #130438] ERROR -- : ["Empty page encountered. Retrying 7 times..."]
E, [2021-06-18T00:27:48.851003 #130438] ERROR -- : ["Empty page encountered. Retrying 8 times..."]
E, [2021-06-18T00:27:50.854114 #130438] ERROR -- : ["Empty page encountered. Retrying 9 times..."]
E, [2021-06-18T00:27:52.857189 #130438] ERROR -- : ["Empty page encountered. Retrying 10 times..."]
E, [2021-06-18T00:27:54.860267 #130438] ERROR -- : ["Empty page 1 of 1 persisted after retrying 10 times. Missing Labels Generation of 1 aborted..."]
E, [2021-06-18T00:27:54.890921 #130438] ERROR -- : Failed, exception: RuntimeError: Empty page 1 of 1 persisted after retrying 10 times. Missing Labels Generation of 1 aborted...
/srv/ontoportal/ncbo_cron/vendor/bundle/ruby/2.6.0/bundler/gems/ontologies_linked_data-de150ab9388c/lib/ontologies_linked_data/models/ontology_submission.rb:577:in `block in loop_classes'
/usr/local/rbenv/versions/2.6.6/lib/ruby/2.6.0/benchmark.rb:308:in `realtime'
/srv/ontoportal/ncbo_cron/vendor/bundle/ruby/2.6.0/bundler/gems/ontologies_linked_data-de150ab9388c/lib/ontologies_linked_data/models/ontology_submission.rb:531:in `loop_classes'
/srv/ontoportal/ncbo_cron/vendor/bundle/ruby/2.6.0/bundler/gems/ontologies_linked_data-de150ab9388c/lib/ontologies_linked_data/models/ontology_submission.rb:1002:in `process_submission'
/srv/ontoportal/ncbo_cron/lib/ncbo_cron/ontology_submission_parser.rb:177:in `process_submission'
bin/ncbo_ontology_process:98:in `block in <main>'
bin/ncbo_ontology_process:81:in `each'
bin/ncbo_ontology_process:81:in `<main>'
[ontoportal@ncbo-stg-app-01 ncbo_cron]$
```
the same errors appears for all UMLS ontologies I have tried

**Comment by alexskr** [comment]
relates to https://github.com/ncbo/goo/issues/111

**Comment by mdorf** [comment]
The resolution of this is to rollback the changes implemented for ncbo/goo#111. They interfere with the proper UMLS ontology processing.

**Comment by graybeal** [comment]
The changes were rolled back and this should be not a problem any more.
